docker/build-local.sh

```diff
@@ -20,6 +20,7 @@ cp -R bin default ${WORK_DIR}/presto-server-${PRESTO_VERSION}
 cp ../presto-cli/target/presto-cli-${PRESTO_VERSION}-executable.jar ${WORK_DIR}
 
 docker build ${WORK_DIR} -f Dockerfile --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}"
+docker buildx build ${WORK_DIR} --platform linux/arm64/v8 -f Dockerfile-arm64v8 --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}-arm64v8" --load
```

Comment:
`buildx` seems to be an experimental feature, so we need to run the command like
```shell
DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build ...